Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10854,6 +10856,24 @@ def xsiam_api_call_with_retries(
     return response
 
 
+def round_up(x):
+    return int(x) + int((x > 0) and (x - int(x)) > 0)
+
+
+def is_zipped_data_size_ideal(zipped_data):
+    return sys.getsizeof(zipped_data) <= XSIAM_EVENT_CHUNK_SIZE
+
+
+def split_xsiam_events_to_chunks(data, size_of_zipped_data):
+    num_of_sub_lists = round_up(size_of_zipped_data / XSIAM_EVENT_CHUNK_SIZE)
+    data_list = str.split(data, '\n') if isinstance(data, str) else data
+    chunk_size = round_up(len(data_list) / num_of_sub_lists)
+    return [
+        data_list[i: i + chunk_size]
+        for i in range(0, len(data_list), chunk_size)
+    ]
+
+
```

Comment:
maybe these functions need to be Internal function like `def events_error_handler(res)`? 
because we won't want that integration could call them, only `send_events_to_xsiam`, no?